aspnetcore/mvc/razor-pages/index.md

```diff
@@ -242,9 +249,9 @@ Add a layout page for the HTML skeleton, and set the `Layout` property from `_Vi
 @{ Layout = "_Layout"; }
 ```
 
-Note that we  placed the layout in the *MyApp/Pages* folder. Pages look for other views (layouts, templates, partials) hierarchically, starting in the same folder as the current page. This means that a layout in the *MyApp/Pages* folder can be used from any page.
+Note that we placed the layout in the *MyApp/Pages* folder. Pages look for other views (layouts, templates, partials) hierarchically, starting in the same folder as the current page. This means that a layout in the *MyApp/Pages* folder can be used from any Razor Page.
```

Comment:
Wouldn't Razor page in reference to a specific page be lowercase page? Razor Pages as a feature is uppercase Page.